opentelekomcloud/resource_opentelekomcloud_compute_instance_v2.go

```diff
@@ -351,20 +350,7 @@ func resourceComputeInstanceV2() *schema.Resource {
 				Optional: true,
 				Default:  false,
 			},
-			"tags": {
-				Type:          schema.TypeSet,
-				Optional:      true,
-				Elem:          &schema.Schema{Type: schema.TypeString},
-				Set:           schema.HashString,
-				ConflictsWith: []string{"tag"},
-				Deprecated:    "Use field tag instead",
-			},
-			"tag": {
-				Type:          schema.TypeMap,
-				Optional:      true,
-				ConflictsWith: []string{"tags"},
-				ValidateFunc:  validateECSTagValue,
-			},
+			"tags": tagsSchema(),
```

Comment:
This is breaking compatibility. `tag` should be deprecated first

Reply:
See https://www.terraform.io/docs/extend/best-practices/deprecations.html#provider-attribute-rename